docs/get-started/Dockerfile.cuda

```diff
@@ -42,13 +43,13 @@ RUN set -ex && \
     echo "auth required pam_wheel.so use_uid" >> /etc/pam.d/su && \
     chgrp root /etc/passwd && chmod ug+rw /etc/passwd
 
-COPY spark-3.0.2-bin-hadoop3.2/jars /opt/spark/jars
-COPY spark-3.0.2-bin-hadoop3.2/bin /opt/spark/bin
-COPY spark-3.0.2-bin-hadoop3.2/sbin /opt/spark/sbin
-COPY spark-3.0.2-bin-hadoop3.2/kubernetes/dockerfiles/spark/entrypoint.sh /opt/
-COPY spark-3.0.2-bin-hadoop3.2/examples /opt/spark/examples
-COPY spark-3.0.2-bin-hadoop3.2/kubernetes/tests /opt/spark/tests
-COPY spark-3.0.2-bin-hadoop3.2/data /opt/spark/data
+COPY spark/jars /opt/spark/jars
+COPY spark/bin /opt/spark/bin
+COPY spark/sbin /opt/spark/sbin
+COPY spark/kubernetes/dockerfiles/spark/entrypoint.sh /opt/
+COPY spark/examples /opt/spark/examples
+COPY spark/kubernetes/tests /opt/spark/tests
+COPY spark/data /opt/spark/data
 
 COPY cudf-21.10.0-SNAPSHOT-cuda11.jar /opt/sparkRapidsPlugin
 COPY rapids-4-spark_2.12-21.10.0-SNAPSHOT.jar /opt/sparkRapidsPlugin
```

Comment:
it looks like these don't match with our instructions either:

Put the 2 jars – rapids-4-spark_<version>.jar, cudf-<version>.jar and getGpusResources.sh in the same directory as spark.

 We could change these COPY commands to use * for the version as well.  Then It think also put a comment in the k8s doc that says edit the file and double check versions match.

Reply:
The [getting-started-kubernetes.md](https://github.com/NVIDIA/spark-rapids/blob/branch-21.10/docs/get-started/getting-started-kubernetes.md]) file has the following text, which I think addresses the concern about double checking that the versions match: 

> Examine the Dockerfile.cuda file to ensure the file names are correct and modify if needed.
>   
> Currently the directory in the local machine should look as below:
> ```shell 
>   $ ls
>   Dockerfile.cuda   cudf-<version>.jar   getGpusResources.sh   rapids-4-spark_<version>.jar   spark
> ```

I updated the COPY command in the dockerfile to include a wildcard.